### PR TITLE
Adds Min's article on kubernetes

### PR DESCRIPTION
**Pull request by jorgelainfiesta**
## Motivation

Min wrote this article in December

## Approach

We opted for a very lightweight generalistic style for this article

[rendered preview](https://deploy-preview-118--frontside.netlify.app/blog/2021-01-29-kubernetes-wet-your-toes/)

### CODE REVIEW

**Comment by minkimcello** [comment]
We should add @taras. He wrote the important parts. And also I didn't know you guys would keep the title. 😆

**Comment by jorgelainfiesta** [comment]
It's your article, I thought that title was like a personality touch that you added, what should it be? (make it water related, I won't make another image for it lol)

Let's not get Taras involved. If you're insecure about some point you can ask Elrick, he wrote an article on Kubernetes earlier on.

File: src/pages/blog/2021-01-29-kubernetes-wet-your-toes.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  - kubernetes
+  - CI/CD
+img: /img/2021-01-kubernetes/hero.png
+---
+
+Kubernetes is perhaps one of the most exciting developments in the DevOps world in the last 15 years. It is an open source implementation of the Borg system that was Google’s “secret sauce” – the thing that allowed them to deploy and manage thousands of services in their clusters. 
+
+With modern cloud PaaS (platform as a service) like Google Cloud and AWS, most frontend and many backend developers can deploy services without learning about the architecture and technologies that power these platforms. In the past, not knowing about these platforms could be easily forgiven because the learning curve was too steep.
+
+Kubernetes changes all that. It makes understanding these platforms approachable and manageable, allowing frontend developers to learn all about them fairly quickly. The architecture of Kubernetes and the approach that the Kubernetes community is taking towards building it continuously lowers the barrier for people who otherwise wouldn’t have an opportunity to learn about Cloud Native Deployment.
```
Comment (minkimcello) [inline src/pages/blog/2021-01-29-kubernetes-wet-your-toes.md]: > The architecture of Kubernetes and the approach that the Kubernetes community is taking towards building it continuously lowers the barrier

Can we reword this to something like `The architecture of Kubernetes and the approach that the community is taking towards building Kubernetes`

File: src/pages/blog/2021-01-29-kubernetes-wet-your-toes.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+Under the hood, Kubernetes consists of a [control panel](https://kubernetes.io/docs/concepts/overview/components/%23control-plane-components) that manages worker [nodes](https://kubernetes.io/docs/concepts/overview/components/%23node-components). The control panel’s API is RESTful, but most of the time you’re not going to be interacting with the API directly. Instead you’ll use the CLI, which talks to the RESTful API. The control panel’s specs are agreed on by the Kubernetes community, but the API definition makes it possible for different implementations of Kubernetes to exist.
+
+In this tutorial, we’ll show you how to set up MicroK8S, which is Ubuntu’s implementation of Kubernetes. It’s small enough to run on your machine and will allow you to play around with Kubernetes without having to pay a cloud provider to host it for you.
+
+## Installation and Start
+
+Let’s go ahead and install MicroK8S for macOS:
+```$ brew install ubuntu/microk8s/microk8s```
```
Comment (minkimcello) [inline src/pages/blog/2021-01-29-kubernetes-wet-your-toes.md]: The back ticks need to be on their own line

File: src/pages/blog/2021-01-29-kubernetes-wet-your-toes.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+```
+$ mkdir ~/.kube && microk8s config > ~/.kube/config
+```
+And now when you start up `k9s` again, you’ll notice the configurations have taken effect. 
+
+If you’ve been following along with all of the previous steps, your `k9s` screen should show the pods that we deployed earlier:
+
+![k9s with pods listed screenshot](/img/2021-01-kubernetes/k9s-listed-pods.png)
+
+What we’re seeing here are the pods from the service we deployed. We’ll be returning to this page very shortly, but for practice if you want to navigate to the list of deployments, press ‘:’ and type in ‘deployments’.
```
Comment (minkimcello) [inline src/pages/blog/2021-01-29-kubernetes-wet-your-toes.md]: Were these meant to be back ticks?

**Review by minkimcello — CHANGES_REQUESTED** [review]

**Review by minkimcello — APPROVED** [review]
👍
